pkg/kubectl/cmd/util/BUILD

```diff
@@ -43,7 +43,6 @@ go_library(
         "//pkg/printers:go_default_library",
         "//pkg/printers/internalversion:go_default_library",
         "//pkg/util/exec:go_default_library",
-        "//pkg/util/i18n:go_default_library",
```

Comment:
Good catch. I think my Bazel files had gotten out of date. They should be fixed now.